addons/website_quote/data/website_quote_demo.xml

```diff
@@ -317,4 +317,288 @@
+            <field name="product_id" ref="product_product_quote_1"/>
+            <field name="product_uom_qty">1</field>
+            <field name="product_uom_id" ref="uom.product_uom_unit"/>
+            <field name="price_unit">12950.00</field>
+            <field name="discount">10.00</field>
+            <field name="website_description" type="html">
+                <section>
+                    <h1>Online Training</h1>
+                </section>
+                <section>
+                    <div class="row">
+                        <div class="col-md-12 mb32">
+                            <p>These courses feature the same high quality course content found in our traditional classroom trainings, supplemented with modular sessions and cloud-based labs. Many of our online learning courses also include dozens of recorded webinars and live sessions by our senior instructors.</p>
+                        </div>
+                        <div class="col-md-offset-1">
+                            <h3>Your advantages</h3>
+                            <ul>
+                                <li>Modular approach applied to the learning method</li>
+                                <li>New interactive learning experience</li>
+                                <li>Lower training budget for the same quality courses</li>
+                                <li>Better comfort to facilitate your learning process</li>
+                            </ul>
+                        </div>
+                        <div class="col-md-12">
+                            <h2>Structure of the Training</h2>
+                        </div>
+                        <div class="col-md-5 col-md-offset-1 mt16 mb16">
+                            <img class="img img-responsive" src="/website_quote/static/src/img/online_training.png" alt="Online training"/>
+                        </div>
+                        <div class="col-md-6 mt32">
+                            <h4><strong>There are three components to the training</strong></h4>
+                            <ul>
+                                <li>Videos with detailed demos</li>
+                                <li>Hands-on exercises and their solutions</li>
+                                <li>Live Q&amp;A sessions with a trainer</li>
+                            </ul>
+                        </div>
+                    </div>
+                </section>
+            </field>
+        </record>
+
+        <record id="website_quote_template_section_line_1b" model="sale.quote.line">
+            <field name="quote_id" ref="website_quote_template_section"/>
+            <field name="name">The very first section</field>
+            <field name="line_type">line_section</field>
+            <field name="sequence">4</field>
+            <field name="website_description" type="html">
+                <span>This is the website_description of a section, I don't know how that works...</span>
```

Comment:
Probably to remove.

To check: don't add silly comments in descriptions, record names, ...